README.md

```diff
@@ -59,14 +59,20 @@ npm install palindrom --save
 Then you can require it CommonJS or ES6/TS style:
 
 ```js
-//CommonJS
-var Palindrom = require('palindrom');
+// CommonJS
+const Palindrom = require('palindrom').Palindrom;
+const PalindromDOM = require('palindrom').PalindromDOM;
 
 // ES6/TS
-import Palindrom from 'palindrom'
+import { Palindrom } from 'palindrom';
+import { PalindromDOM } from 'palindrom';
```

Comment:
Why not `import { Palindrom, PalindromDOM} from 'palindrom';`?

Reply:
In my mind these are two different examples, all you ever need is either Palindrom or PalindromDOM. Should I present it better?

Reply:
no, it's fine as it is, that was just a question.